src/txn/txn_rollback_to_stable.c

```diff
@@ -201,66 +201,69 @@ __txn_abort_newer_row_leaf(
     }
 }
 
+/*
+ * __txn_page_needs_rollback --
+ *     Check whether the page needs has modification newer than the given timestamp.
```

Comment:
Looks like some words got dropped from this comment. I'd prefer something clearer like:
"Check whether the page needs rollback. Return true if ... . Otherwise return false."

Reply:
Used the provided sentence update and updated accordingly.